resource-watch/resource-watch
# Widget creation/edition: remove an error from the console

## Overview
When the textarea of the advanced mode is empty, the  preview fails and displays an error in the console. To prevent this, the  preview is only shown if the widget config contains a "data" object.

## Testing instructions
To test this PR, go to http://localhost:3000/admin/data/widgets/new, select a dataset and choose the advanced mode. You shouldn't see any error related to "data is undefined" in the console.

## Pivotal task
None.
